Dockerfile

```diff
@@ -0,0 +1,18 @@
+# public docker image
```

Comment:
I"m not sure you want to commit this docker image

Reply:
I can remove it, no problem.